packages/tql/src/dialects/postgres.ts

```diff
@@ -0,0 +1,104 @@
+import { IdenticalColumnValidator } from '../values-object-validator';
+import type { DialectImpl } from '../types';
+import { BaseDialect } from '../dialect';
+import {
+  type TqlIdentifiers,
+  type TqlList,
+  type TqlParameter,
+  type TqlTemplateString,
+  type TqlSet,
+  type TqlValues,
+} from '../nodes.js';
+
+export class PostgresDialect extends BaseDialect implements DialectImpl {
+  string(str: TqlTemplateString): void {
+    this.appendToQuery(str.value);
+  }
+
+  parameter(param: TqlParameter): void {
+    const paramNumber = this.appendToParams(param.value);
+    this.appendToQuery(`$${paramNumber}`);
+  }
+
+  identifiers(ids: TqlIdentifiers): void {
+    if (Array.isArray(ids.values)) {
+      this.appendToQuery(
+        ids.values.map((v) => PostgresDialect.escapeIdentifier(v)).join(', '),
+      );
+    } else {
+      this.appendToQuery(PostgresDialect.escapeIdentifier(ids.values));
+    }
+  }
+
+  list(vals: TqlList): void {
+    this.appendToQuery('(');
+    const queryItems: string[] = [];
+    for (const param of vals.values) {
+      const paramNumber = this.appendToParams(param);
+      queryItems.push(`$${paramNumber}`);
+    }
+    this.appendToQuery(queryItems.join(', '));
+    this.appendToQuery(')');
+  }
+
+  values(entries: TqlValues): void {
+    if (Array.isArray(entries.values)) {
+      // it's multiple entries
+      const validator = new IdenticalColumnValidator();
+      let first = true;
+      let columns: string[] = [];
+      const rows: string[] = [];
+      for (const entry of entries.values) {
+        validator.validate(entry);
+        if (first) {
+          first = false;
+          columns = Object.keys(entry);
+          this.appendToQuery(
+            `(${columns
+              // eslint-disable-next-line @typescript-eslint/unbound-method -- this rule is idiotic, this is a static method
+              .map(PostgresDialect.escapeIdentifier)
```

Comment:
I agree this rule is unnecessary but maybe for the sake of clean code we can be a little more verbose?

```suggestion
              .map(column => PostgresDialect.escapeIdentifier(column))
```

(same with down below)

Reply:
The point at which ESLint makes clean code less clean to avoid disabling the rule making the clean code error is the point at which ESLint has outlived its usefulness... <picture data-single-emoji=":meow_eyeroll:" title=":meow_eyeroll:"><img class="emoji" width="20" height="auto" src="https://emoji.slack-edge.com/T0CAQ00TU/meow_eyeroll/235022ca4b2f9020.gif" alt=":meow_eyeroll:" align="absmiddle"></picture> 

Fixed, begrudgingly! <picture data-single-emoji=":lol:" title=":lol:"><img class="emoji" width="20" height="auto" src="https://emoji.slack-edge.com/T0CAQ00TU/lol/5986e156eca2154f.png" alt=":lol:" align="absmiddle"></picture>